arch/arm/core/aarch32/Kconfig

```diff
@@ -215,6 +215,34 @@ config ARM_NONSECURE_FIRMWARE
 	  resources of the Cortex-M MCU, and, therefore, it shall avoid
 	  accessing them.
 
+config ARM_STORE_EXC_RETURN
+	bool
+	default y if USERSPACE || FPU_SHARING || ARCH_HAS_TRUSTED_EXECUTION
```

Comment:
Why not ARM_NONSECURE_FIRMWARE @oyvindronningstad ?
ARCH_HAS does not mean anything, until users build with the feature.....